internal/utils.go

```diff
@@ -178,6 +178,12 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]interface{
 		extraDeps := policyConf.Manifests[i].ExtraDependencies
 
 		for _, manifest := range manifestGroup {
+			err := setGatekeeperEnforcementAction(manifest,
+				policyConf.Manifests[i].GatekeeperEnforcementAction)
+			if err != nil {
+				return nil, err
```

Comment:
Could you provide more context in the return error such as the manifest index and that this error occurred due to setting `spec.enforcementAction`?